library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -871,12 +871,16 @@ public void notifyStateChangedToLoaded(Section section, State previousState) {
             }
         }
 
-        notifyItemChangedInSection(section, 0);
-
         int contentItemsTotal = section.getContentItemsTotal();
 
-        if (contentItemsTotal > 1) {
-            notifyItemRangeInsertedInSection(section, 1, contentItemsTotal - 1);
+        if (contentItemsTotal == 0) {
+            notifyItemRemovedFromSection(section, 0);
```

Comment:
@luizgrp 

Hi, sorry, I don't understand "developer should call the correct notify method". What is the correct notify method?

    section.setState(LOADING);

    //
    // (Some network operation).
    //

    section.setData(new ArrayList<>());
    // Instead of marked as EMPTY state, developer decides it is more suitable to mark it as LOADED state.
    section.setState(LOADED):

    // I think it make sense for developer to call notifyStateChangedToLoaded right?
    // If it is so, isn't it is suitable for us to help developer call notifyItemRemovedFromSection ?
    notifyStateChangedToLoaded(section, LOADING);